plugins/plugin-webpack/README.md

```diff
@@ -14,16 +14,30 @@ Add `@snowpack/plugin-webpack` to `snowpack.config.json`:
 
 ```json
 {
-  "plugins": [["@snowpack/plugin-webpack", { /* see "Plugin Options" below */ }]]
+  "plugins": [
+    [
+      "@snowpack/plugin-webpack",
+      {
+        /* see "Plugin Options" below */
+      }
+    ]
+  ]
 }
 ```
 
 or to `snowpack.config.js`:
 
 ```js
 module.exports = {
-  plugins: [["@snowpack/plugin-webpack", { /* see "Plugin Options" below */ }]]
-}
+  plugins: [
```

Comment:
Yay. Better docs on top of my changes! :smile:

Reply:
oh my I'm sorry for the unrelated changes. I've `prettier` setup to format on save, and didn't pay attention.

Reply:
No no, I think it's fine :)